test/fixtures/test-partials2/partials/pod.yaml.erb

```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: web-from-partial
+spec:
+  replicas: 1
+  template:
```

Comment:
Nitpick: only this part is a podspec, despite the name of the file.

Reply:
The goal of the test is to ensure that it's not loading the identically named partial in the other directory, so I kept the partial name the same but ensured the render would fail to validate if it grabbed the wrong partial